src/transformers/models/layoutlmv2/modeling_layoutlmv2.py

```diff
@@ -821,24 +821,43 @@ def forward(
         return_dict: Optional[bool] = None,
     ) -> Union[Tuple, BaseModelOutputWithPooling]:
         r"""
-        Returns:
+        Return:
 
         Examples:
 
-        ```python
-        >>> from transformers import LayoutLMv2Processor, LayoutLMv2Model
-        >>> from PIL import Image
+        LayoutLMv2 depends on detectron2, torchvision and tesseract; passing doctests requires their installation. Run
+        the following to install them: `python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'`
+        `python -m pip install torchvision tesseract`
+
+
+            ```python
+            >>> from transformers import LayoutLMv2Processor, LayoutLMv2Model, set_seed
+            >>> from PIL import Image
+            >>> import torch
+            >>> from datasets import load_dataset
+
+            >>> set_seed(88)
+
+            >>> processor = LayoutLMv2Processor.from_pretrained("microsoft/layoutlmv2-base-uncased")
+            >>> model = LayoutLMv2Model.from_pretrained("microsoft/layoutlmv2-base-uncased")
+
 
-        >>> processor = LayoutLMv2Processor.from_pretrained("microsoft/layoutlmv2-base-uncased")
-        >>> model = LayoutLMv2Model.from_pretrained("microsoft/layoutlmv2-base-uncased")
+            >>> dataset = load_dataset("hf-internal-testing/fixtures_docvqa")
+            >>> image_path = dataset["test"][0]["file"]
+            >>> image = Image.open(image_path).convert("RGB")
 
-        >>> image = Image.open("name_of_your_document - can be a png file, pdf, etc.").convert("RGB")
+            >>> encoding = processor(image, return_tensors="pt")
 
-        >>> encoding = processor(image, return_tensors="pt")
+            >>> outputs = model(**encoding)
+            >>> last_hidden_states = outputs.last_hidden_state
 
-        >>> outputs = model(**encoding)
-        >>> last_hidden_states = outputs.last_hidden_state
-        ```"""
+            >>> last_hidden_states.shape
+            torch.Size([1, 342, 768])
+
+            >>> round(torch.sum(last_hidden_states).item(), 2)
+            -3139.01
```

Comment:
I got

```
939             >>> round(torch.sum(last_hidden_states).item(), 2)
Expected:
    -3139.01
Got:
    -3140.06
```

It's unclear to me if this is caused by the different machines, package versions, etc. But it was deterministic on the Linux machine I ran the tests.

Reply:
Yes, this won't be something that can be useful in tests as it accumulates all the small errors, so won't be reproducible on different machines/hardware.